asserts/asserts.go

```diff
@@ -103,7 +103,10 @@ var maxSupportedFormat = map[string]int{}
 
 func init() {
 	// register maxSupportedFormats while breaking initialisation loop
-	maxSupportedFormat[SnapDeclarationType.Name] = 1
+
+	// 1: plugs and slots
+	// 2: support for $SLOT()/$PLUG()/$MISSING
```

Comment:
So $MISSING is for stating that the attribute must not be set at all? What would be an example use case of that? auto-connect only if an attribute is not set? If so, is $MISSING the right word; would $DEFAULT be better?